bio/galah/wrapper.py

```diff
@@ -0,0 +1,42 @@
+__author__ = "Filipe G. Vieira"
+__copyright__ = "Copyright 2023, Filipe G. Vieira"
+__license__ = "MIT"
+
+from snakemake.shell import shell
+
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+extra = snakemake.params.get("extra", "")
+
+
+fas = snakemake.input.get("fas", "")
+if fas:
+    fas = " ".join(fas)
+    fas = f"--genome-fasta-files {fas}"
```

Comment:
I guess this is a matter of style, so feel free to accept or discard:
```suggestion
    fas = f"--genome-fasta-files {' '.join(fas)}"
```